code/game/atoms_movable.dm

```diff
@@ -148,17 +148,31 @@
 	return 1
 
 /mob/living/forceMove(atom/destination)
-	stop_pulling()
 	if(buckled)
-		buckled.unbuckle_mob(src,force=1)
-	// in lieu of "unbuckle_all_mobs"
+		addtimer(src, "checkBuckled", 1, TRUE)
 	if(buckled_mob)
-		unbuckle_mob(buckled_mob,force=1)
+		addtimer(src, "checkBuckledMob", 1, TRUE)

```

Comment:
No need for a second proc here...
`addtimer(buckled_mob, "checkBuckled", 1, TRUE)`